tests/evidence_string_generation/test_evidence_strings.py

```diff
@@ -13,288 +12,51 @@
+        self.maxDiff = None  # this is required to display evidence string diffs (if any)
+        self.test_args = get_input_data_for_evidence_string_generation()
 
-    report = clinvar_to_evidence_strings.Report()
+    def test_genetics_evidence_string(self):
+        """Verifies expected genetics evidence string generation."""
+        evidence_string = json.dumps(evidence_strings.CTTVGeneticsEvidenceString(*self.test_args),
+                                     sort_keys=True, indent=2)
+        expected_evidence_string = open(config.expected_genetics_evidence_string).read()
+        self.assertEqual(evidence_string, expected_evidence_string)
```

Comment:
per convention, expected value comes first...

Reply:
It's not a convention, it's a JUnit API design (that always seemed unintuitive to me). The [api for python's unittest](https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.assertEqual) just says "`first` and `second` should be equal". I'm not sure it brings any benefit to swap all the assertions in these files, specially when the expected parameter is prefixed with "expected_".

Reply:
Unittest does not enforce strict requirements in its API; however, the basic example in the official documentation uses (actual, expected) for ordering: https://docs.python.org/3/library/unittest.html#basic-example